bread/static/design/carbon_design/scss/styles.scss

```diff
@@ -149,3 +149,21 @@ fieldset[readonly=true] :disabled {
 .highlighted {
     font-weight: 600;
 }
+
```

Comment:
Yes, I think totally justified, nice and generic implementation for the icon-feature.